include/sharg/config.hpp

```diff
@@ -59,6 +60,18 @@ struct config
     //!\brief The description to be shown on any (exported) help page.
     std::string description{};
 
+    /*!\brief The default message to be shown on any (exported) help page.
+     *
+     * "Default: " will be prepended. "." will be appended.
+     *
+     * ### Example
+     *
+     * A default_message of "Same as --i", will result in "Default: Same as --i.".
+     *
```

Comment:
```suggestion
     *
     * `parser.add_option(j, sharg::config{.short_id = 'j'. default_message = "Same as -i"})`
     * will result in the help page looking like this:
     * ```
     * -j (signed 32 bit integer)
     *     Default: Same as -i.
     * ```
     *
```